.gitignore

```diff
@@ -4,6 +4,9 @@ docs/images/supporters
 docs/api
 mocha.js
 mocha.js.map
+esm/mocha.js
+esm/mocha.js.map
+esm/package.json
```

Comment:
Oh 😬 I didn't think adding support for ESM would require an entirely new output file. This file is pretty huge: 600k+ characters, 620K on disk. We can't add such a large addition to package size for all users.

I had assumed the entry point would be a minimal shim, i.e.:

```js
import "./mocha.js"

const { mocha } = globalThis;

const describe = mocha.describe.bind(mocha);
const it = mocha.it.bind(mocha);
const setup = mocha.setup.bind(mocha);

export { describe, it, setup };
```

Is that not doable for some reason?

Reply:
That makes sense. I'm unfortunately dealing with some health problems, so I think I will just nix this PR.